geomstats/_backend/pytorch/__init__.py

```diff
@@ -787,3 +791,16 @@ def triu_to_vec(x, k=0):
     n = x.shape[-1]
     rows, cols = triu_indices(n, k=k)
     return x[..., rows, cols]
+
+
+def mat_from_diag_triu_tril(diag, tri_upp, tri_low):
+    """build matrices (or batch of matrices) from diag and
```

Comment:
NIT: The docstring needs to follow our docstring conventions

Reply:
> NIT: The docstring needs to follow our docstring conventions

Done